static/app/views/settings/account/apiTokenRow.spec.tsx

```diff
@@ -16,7 +16,7 @@ describe('ApiTokenRow', () => {
     const cb = jest.fn();
     render(<ApiTokenRow onRemove={cb} token={TestStubs.ApiToken()} />);
 
-    await userEvent.click(screen.getByRole('button'));
+    await userEvent.click(screen.getByLabelText('Remove'));
```

Comment:
there are two buttons in the component now, so we have to select a different way.